nltk/tokenize/nist.py

```diff
@@ -55,6 +82,35 @@ class NISTTokenizer(TokenizerI):
     LANG_DEPENDENT_REGEXES = [PUNCT, PERIOD_COMMA_PRECEED,
                               PERIOD_COMMA_FOLLOW, DASH_PRECEED_DIGIT]
 
+    # Perluniprops characters used in NIST tokenizer.
+    Number = text_type(''.join(set(perluniprops.chars('Number')))) # i.e. \p{N}
```

Comment:
Unusual to have uppercase variable name?

Reply:
Using lowercase, those were bad habits ported from the perl script ;P